_protected/app/langs/fr_CA/config/config.ini

```diff
@@ -0,0 +1,27 @@
+; INFORMATION DE LA LANGUE
+
+[language]
+
+; Pour la langue française ne touchez à rien !
+; Le préfixe de la langue : Français = fr, English = en
+; Vous pouvez également être plus précis. EX : Français en Belgique = fr-BE, Français en France = fr-FR, English aux États-Unis = en-US, etc.
+lang = fr-CA
+
+; ltr = Langue de gauche à droite (Français, English, ...) | rtl = Langue de droite à gauche (Arabe, Hébreu, ...).
+dir = ltr
+
+; Encodage pour la langue : En général UTF-8 fonctionne toujours.
+charset = "utf-8"
+
+[language.information]
+name = Français Canadien
+author = "Polyna-Maude R.-Summerside"
+email = "polynamaude@gmail.com"
+website = "http://ph7cms.com"
+
+[language.application]
+timezone = "Europe/Paris"
```

Comment:
I think, this should be changed. Maybe "America/New_York" (also UTC -4), since "America/Montreal" isn't available.
List: http://php.net/manual/en/timezones.america.php

Reply:
I will check between America/Toronto and America/Ottawa. I think we have a different daylight saving time from New York state.

I'll correct those.

Do you want me to push a translated language.php for French France?

Reply:
1. Yes, I think you are right concerning the DST 👍 
"America/Toronto" sounds the best, since I cannot find America/Ottawa [in the list](https://secure.php.net/manual/en/timezones.america.php).

2. If you have time, that would be great if you could create another PR for the fr_FR translation with the `language.php` translated.

Reply:
Hi @polynamaude Just let me know when the few changes from https://github.com/pH7Software/pH7-Internationalization/pull/11#pullrequestreview-141095344 is done, so I can merge it 😄